File: 1017.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var value = input.split("\n");

// URI 1017 | GASTO DE COMBUSTÍVEL

var tempoViagem = parseInt(value.shift());
var velocidadeMedia = parseInt(value.shift());

var distanciaPercorrida = tempoViagem * velocidadeMedia;
var litrosNecessarios = (distanciaPercorrida / 12).toFixed(3);

console.log(litrosNecessarios);

File: Python/1017.py
# BEE 1017 | GASTO DE COMBUSTÍVEL

tempoViagem = int(input())
velocidadeMedia = int(input())

distanciaPercorrida = tempoViagem * velocidadeMedia
qtdLitros = distanciaPercorrida / 12
print("%0.3f" % qtdLitros)